specs/src/rationale/message_block_layout.md

```diff
@@ -35,14 +35,13 @@ This, however, requires the block producer to interact with the transaction send
 
 ### Non-Interactive Default Rules
 
-As a non-consensus-critical default, we can impose some additional rules on message placement to make the possible starting locations of messages sufficiently predictable and constrained such that users can deterministically compute subtree roots without interaction:
+As a non-consensus-critical default, we can impose one additional rule on message placement to make the possible starting locations of messages sufficiently predictable and constrained such that users can deterministically compute subtree roots without interaction:
 
-1. Messages that span multiple rows must begin at the start of a row (this can occur if a message is longer than `k` shares _or_ if the block producer decides to start a message partway through a row and it cannot fit).
-1. Messages begin at a location aligned with the largest power of 2 that is not larger than the message length or `k`.
+> Messages start at an index that is a multiple of the message minimum square size. The message minimum square size is the smallest square that can contain the message.
```

Comment:
Can you add a line that "contain" here ignores other transactions and messages, and only considers the message in isolation? Something like that.

Reply:
Yes absolutely. How about

> Messages start at an index that is a multiple of the message minimum square size. The message minimum square size is the smallest square that can contain the message in isolation (i.e. a square with only this message and no other transactions or messages).

Reply:
Yeah sure.

Reply:
Resolved in https://github.com/celestiaorg/celestia-app/pull/1156/commits/8a970ab656b740b8cf67878fff9a1d6940efa33d